app/server/utils/metrics/metric.py

```diff
@@ -17,14 +17,18 @@ def execute_query(self, user_filters: dict = None, date_filters_dict=None, enabl
              aggregated_query and total_query
         """
         actions = { 'query': self.query_actions, 'aggregated_query': self.aggregated_query_actions, 'total_query': self.total_query_actions }
+
+        # Build the dict of queries to execute. Ungrouped metrics don't have aggregated queries,
+        # and sometimes we only want aggregates and totals (based on calculate_only_aggregates)
         if self.is_timeseries:
             if calculate_only_aggregates:
```

Comment:
should this be `dont_include_timeseries` given that we now have `if self.aggregated_query:` below?